.github/workflows/update-artifacts.yml

```diff
@@ -75,3 +81,26 @@ jobs:
             else
               echo "No changes to commit"
             fi
+        
+        - name: Set up Docker Buildx
+          uses: docker/setup-buildx-action@v2
+
+        - name: Log in to JFrog Artifactory private
+          uses: docker/login-action@v2
+          with:
+            username: github-ci
+            password: ${{ secrets.JFROG_REGISTRY_PASSWORD }}
+            registry: tfy.jfrog.io
+
+        - name: Upload Artifacts
+          run: |
+            python scripts/upload-artifacts/upload_artifacts.py \
+              helm \
+              charts/tfy-k8s-aws-eks-inframold/artifacts-manifest.json \
```

Comment:
Won't using only one inframold chart exclude some images/charts that are not in tfy-k8s-aws-eks-inframold artifacts-manifest.json file?